src/name_service/delivery/service.clj

```diff
@@ -3,11 +3,13 @@
             [io.pedestal.http.body-params :as body-params]
             [name-service.delivery.use-case :as uc]))
 
-(def routes `{"/" {:interceptors [(body-params/body-params) http/html-body]
-                   :get uc/get-binding
-                   :post uc/post-binding
-                   :patch uc/patch-binding
-                   :delete uc/delete-binding}})
+
+(def common-interceptors [(body-params/body-params) http/html-body])
+
+(def routes #{["/" :get (conj common-interceptors `uc/get-binding)]
+              ["/" :post (conj common-interceptors `uc/post-binding)]
+              ["/" :patch (conj common-interceptors `uc/patch-binding)]
+              ["/" :delete (conj common-interceptors `uc/delete-binding)]})
```

Comment:
Hm I'm thinking. Wouldn't it be better to modify the default interceptors instead of adding those everytime to the routes?
http://pedestal.io/reference/default-interceptors

Reply:
Good idea 👍